recipes/isce3/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "isce3" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/isce-framework/isce3/archive/v{{ version }}.tar.gz
+  sha256: 9de6a6839818f2553e2f31e49ca5335a3c41b77e4482777e17f376d131da589b
+  patches:
+    - fix-find-numpy.patch
+    - fix-missing-includes.patch
+
+build:
+  number: 0
+  skip: True  # [win or py<36]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - cython
+    - gtest
+    - ninja
+  host:
+    - cereal
+    - eigen
+    - fftw
+    - gdal
+    - h5py
+    - numpy {{ numpy }}
+    - pybind11
+    - python
+    - pip
+    - pyre
+    - pytest
+    - ruamel.yaml
+    - yamale
```

Comment:
Have you checked if these are all needed in `host`. They might only be needed in `run`.